metadata/workflowhub.yml

```diff
@@ -103,7 +103,8 @@ genome-annotation/annotation-with-prokka/galaxy-workflow-workflow-constructed-fr
 genome-annotation/apollo/main-workflow: '1554'
 genome-annotation/bacterial-genome-annotation/bacterial-genome-annotation: '1513'
 genome-annotation/braker3/braker: '1509'
-genome-annotation/comparison-braker-helixer-annotation/galaxy-workflow-comparison-of-two-annotation-tools--helixer-and-braker3: '1746'
+? genome-annotation/comparison-braker-helixer-annotation/galaxy-workflow-comparison-of-two-annotation-tools--helixer-and-braker3
```

Comment:
that looks strange

Reply:
hmm, I saw the same in a different PR and fixed it, but I guess it will keep trying to make this change, will have to try to figure out why..